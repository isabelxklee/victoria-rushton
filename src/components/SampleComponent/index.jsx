import React from 'react'

const SampleComponent = ({children}) => (
  <span>
    <h1>hello world</h1>
    <p>{children}</p>
  </span>
)

export default SampleComponent
