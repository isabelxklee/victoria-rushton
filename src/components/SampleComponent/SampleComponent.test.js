import React from 'react'
import renderer from 'react-test-renderer'
import SampleComponent from './index.jsx'

describe('SampleComponent', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<SampleComponent>helllo worlddd</SampleComponent>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
