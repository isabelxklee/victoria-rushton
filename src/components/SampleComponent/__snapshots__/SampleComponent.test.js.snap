// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleComponent snapshot renders 1`] = `
<span>
  <h1>
    hello world
  </h1>
  <p>
    helllo worlddd
  </p>
</span>
`;
